CI/travis/lib.sh

```diff
@@ -278,7 +278,7 @@ run_docker_script() {
 	local DOCKER_IMAGE="$2"
 	local OS_TYPE="$3"
 	local MOUNTPOINT="${4:-docker_build_dir}"
-	sudo docker run --rm=true \
+	sudo docker run --rm=true -e "TRAVIS=${TRAVIS}" \
```

Comment:
specifically for this env-var passing, i did PR: https://github.com/analogdevicesinc/libiio/pull/420

this is a bit more flexible;
once you need an env-var in a docker, it's opening the gateway to more env-vars;
especially if you want to move more stuff into dockers [when wanting newer or weirder] images;